### PR TITLE
added dockerfile. removed testing environment

### PR DESCRIPTION
**Pull request by Lassejoe**
Added a dockerfile and included a docker command to run the docker image; removed the previous installation guide.
Removed all testing from the documentation.

### CODE REVIEW

**Review by ct-clearhaus — APPROVED** [review]
Nice!

Just one well-meant point I cannot hold back: https://chris.beams.io/posts/git-commit/#imperative 🙂

**Comment by Lassejoe** [comment]
@ct-clearhaus after the merge will the site auto deploy the changes?

**Comment by ct-clearhaus** [comment]
> auto deploy

Nah. It runs in heroku IIRC, and @soltveit has access — that's why I assigned this PR to him 🙂

**Comment by soltveit** [comment]
Just checked my Heroku account and nothing there related to the MPI. Only `docs.gateway.clearhaus.com` stuff.

Perhaps it's time to migrate `docs.3dsecure.io` to AWS?

**Comment by davidfromdk** [comment]
I've created this https://github.com/clearhaus/issues-tech-skunk/issues/26. Think we should just move both sites while we are at it. Any objections from your side regarding the `
docs.gateway.clearhaus.com`, @ct-clearhaus  @soltveit ?

**Comment by soltveit** [comment]
No objections from my side. @ct-clearhaus ?

File: README.md [hunk 1]
```diff
@@ -5,26 +5,17 @@
...
-```
-or
-```
-brew install node
-```
+### Installation
 
-### Install bower
+```bash
+git clone https://github.com/clearhaus/gateway-api-docs.git
```
Comment (jsmpereira) [inline README.md:12]: @Lassejoe Just noticed that this should've been:

```suggestion
git clone https://github.com/3dsecure/mpi-docs.git
```

**Pull request merged**